packages/fxa-settings/src/react-app-env.d.ts

```diff
@@ -55,7 +55,7 @@ declare module '*.webp' {
 }
 
 declare module '*.svg' {
-  import * as React from 'react';
+  import React from 'react';
```

Comment:
Ohh, was this the culprit of some of our broken SVGs?

Reply:
This, plus there was an error in one of the image component names `EmailCode` vs `EmailCodeImage`